go.mod

```diff
@@ -39,8 +39,8 @@ require (
 	github.com/stretchr/testify v1.8.2
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.24.0
-	go.mongodb.org/mongo-driver v1.11.3
+	go.mongodb.org/atlas v0.23.2-0.20230328111225-e40463c98c4e
+	go.mongodb.org/mongo-driver v1.11.2
```

Comment:
this was just updated on master
`go get -u go.mongodb.org/mongo-drive && go mod tidy` and commit those changes